tests/routes/pro/get_offerers_names_test.py

```diff
@@ -8,6 +8,28 @@
 
 
 class Returns200:
+    @pytest.mark.usefixtures("db_session")
+    def should_returns_all_offerers_managed_by_user_when_user_is_not_admin(self, app):
+        # given
+        pro = users_factories.UserFactory(isBeneficiary=False)
+        owned_offerer_validated = offers_factories.OffererFactory()
+        owned_offerer_not_validated = offers_factories.OffererFactory(validationToken="token")
+        other_offerer = offers_factories.OffererFactory()
+        offers_factories.UserOffererFactory(user=pro, offerer=owned_offerer_validated)
+        offers_factories.UserOffererFactory(user=pro, offerer=owned_offerer_not_validated)
+
+        # when
+        response = TestClient(app.test_client()).with_auth(pro.email).get("/offerers/names?validated=false")
+
+        # then
+        assert response.status_code == 200
+        assert len(response.json) == 1
+
+        offerer_ids = [offererName["id"] for offererName in response.json["offerersNames"]]
+        assert humanize(owned_offerer_validated.id) in offerer_ids
+        assert humanize(owned_offerer_not_validated.id) in offerer_ids
+        assert [humanize(other_offerer.id)] not in offerer_ids
+
     @pytest.mark.usefixtures("db_session")
     def should_returns_only_validated_offerers_managed_by_user_when_user_is_not_admin(self, app):
```

Comment:
```suggestion
    def should_returns_only_validated_offerers_managed_by_user_when_user_is_not_admin_and_request_validated_only(self, app):
```

Reply:
```suggestion
    def test_not_admin_get_offerers_without_filters(self, app):
```
ca me semble plus lisible.

Reply:
Je trouve cela dommage car on perd ce qu'on attend comme résultat dans le nom du test
Mais si tu préfères, alors plutôt `pro_user` que `not_admin`